change/@internal-react-composites-66227eba-9d43-4ef3-a745-88982b43fc9c.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Exposed a prop  in SendBox to allow sendbox to render file uploads without implementing onRenderFileUploads",
```

Comment:
This changelog is misleading: How about "use default file upload UI from SendBox".